### PR TITLE
Correct ad slot sizes

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Correct the different ad sizes for different ad slots to match the current values in front end. This is the first of two PRs, in the next one I am going to refactor the code to match closer the logic that has been introduced in frontend recently

### CODE REVIEW

File: src/model/advertisement.ts [hunk 1]
```diff
@@ -31,7 +33,13 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'mostpop',
```
Comment (oliverlloyd) [inline src/model/advertisement.ts]: I noticed that [this AdSlot.scala file](https://github.com/guardian/frontend/blob/04a2b9112c5bf792510965c405cbd406b083288e/common/app/common/commercial/AdSlot.scala) has different values in it - how do we know which ones are right?

File: src/model/advertisement.ts [hunk 2]
```diff
@@ -19,7 +21,7 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'top-above-nav',
```
Comment (oliverlloyd) [inline src/model/advertisement.ts]: For top-above-nav, there's also [Commercial.scala](https://github.com/guardian/frontend/blob/b6cb79e726432767b68ff22ddfc3e5fd3392a1c2/common/app/views/support/Commercial.scala#L128) which seems to have sizing code in it - not sure which is right 🤷‍♂

File: src/model/advertisement.ts [hunk 3]
```diff
@@ -19,7 +120,17 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
...
             adTypes: ['mpu-banner-ad', 'rendered'],
             sizeMapping: {
-                tablet: ['1,1|2,2|728,90|88,71|fluid'],
+                // mirror frontend file mark: 432b3a46-90c1-4573-90d3-2400b51af8d0
```
Comment (liywjl) [inline src/model/advertisement.ts]: what is a file mark??

File: src/model/advertisement.ts (hunk 1 above)
Comment (shtukas) [reply src/model/advertisement.ts]: Going to ask Ioanna

File: src/model/advertisement.ts (hunk 2 above)
Comment (shtukas) [reply src/model/advertisement.ts]: Done

File: src/model/advertisement.ts (hunk 3 above)
Comment (shtukas) [reply src/model/advertisement.ts]: Ids that I use to refers to files or parts of a file from other places (in the same project or other projects) :)

**Pull request merged**